addons/common/functions/fnc_getInPosition.sqf

```diff
@@ -18,10 +18,10 @@
 #define CANGETINDRIVER      (isNull (driver _vehicle)             || {!alive driver _vehicle})               && {!lockedDriver _vehicle}           && {getNumber (_config >> "isUav") != 1}
 #define CANGETINTURRETINDEX (isNull (_vehicle turretUnit _turret) || {!alive (_vehicle turretUnit _turret)}) && {!(_vehicle lockedTurret _turret)} && {getNumber (_config >> "isUav") != 1}
 
-private ["_unit", "_vehicle", "_position", "_index"];
+private ["_position", "_index"];
```

Comment:
This function needs to be reviewed carefully. Github is being stupid about the diff